packages/react/src/components/Masthead/README.md

```diff
@@ -66,12 +77,12 @@ A cors proxy can be configured using the following
 
 `CORS_PROXY=https://myproxy.com/`
 
-## 🙌 Contributing
+## í ½í¹Œ Contributing
 
 We're always looking for contributors to help us fix bugs, build new features,
 or help us improve the project documentation. If you're interested, definitely
-check out our [Contributing Guide](/.github/CONTRIBUTING.md)! 👀
+check out our [Contributing Guide](/.github/CONTRIBUTING.md)! í ½í±€
 
-## 📝 License
+## í ½í³ License
```

Comment:
oh no the emojis

Reply:
@annawen1 haha yeah emacs is a real pain when it comes to emojis :(